components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,39 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				gap: 12px;
+				height: var(--d2l-sortable-button-height);
 				letter-spacing: inherit;
 				margin: 0;
-				padding: 0;
+				padding: 0.75rem;
 				text-decoration: none;
+				width: 100%;
 			}
 			button::-moz-focus-inner {
 				border: 0;
 			}
 			button:disabled {
 				opacity: 0.5;
 			}
-			button:hover {
-				text-decoration: underline;
-			}
 			button:focus-visible,
 			button:${unsafeCSS(getFocusPseudoClass())} {
-				border-radius: 0.2rem;
+				border-radius: var(--d2l-sortable-button-border-radius);
 				box-shadow: 0 0 0 2px #ffffff, 0 0 0 4px var(--d2l-color-celestine);
+				height: var(--d2l-sortable-button-focus-height);
+				margin-inline-start: 0.2rem;
 				outline-style: none;
+				padding-inline-start: 0.55rem;
```

Comment:
Where did these numbers come from for this and `margin-inline-start`? Is it possible to have a calculation which makes their values more clear, or maybe a small comment that specifies how you came to them?